acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  BowlOrderDetailViewController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+protocol DidDeselectOption: class {
+    func deselectOption(at index: Int)
+}
+
+protocol DidSelectOption: class {
```

Comment:
This should be one delegate protocol with deselect. Ideally would have delegate in the name.